### PR TITLE
Milestone 2: Finished Mobile version

### PR DESCRIPTION
**Pull request by prg-04**
## In this Milestone, I have done the following step to finish setting up a "My-Portfolio" repository:

- [ ] Add package-lock.json file to `.gitignore`

- [ ] Finish HTML markups

- [ ] Make sure that any files that are not relevant are ignored by git.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,40 +13,219 @@
...
+          <div class="c-skills">
+            <h3>&#9679; Skills</h3>
+            <ul>
+              <li>Codekit</li>
+              <li>Github</li>
+              <li>Codepen</li>
+              <li>Gitlab</li>
+              <li>Terminal</li>
+            </ul>
+          </div>
```
Comment (zainsadaqat) [inline index.html:226]: - [ ] Please use the required color and alignment for this section. 👍 
### [YOUR DESIGN]
Screenshot 2022-08-27 at 1 06 37 PM
### [REQUIRED DESIGN]
Screenshot 2022-08-27 at 1 07 31 PM

File: index.html [hunk 2]
```diff
@@ -13,40 +13,219 @@
...
+        <div class="about-me">
+          <h2>About Me</h2>
+          <p>
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hesitate to contact me.
+          </p>
+          <button>Get My Resume</button>
+        </div>
```
Comment (zainsadaqat) [inline index.html:195]: - [ ] Please fix the about section by increasing the width for the description and the button. 👍🏻 
#### [REQUIRED DESIGN]
Screenshot 2022-08-27 at 1 04 35 PM
#### [YOUR DESIGN]
Screenshot 2022-08-27 at 1 05 25 PM

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# Required Changes 🔁
# Hello @prg-04  😄 

You're doing a great job on your project. We're proud of you. Good job so far! 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
  ✅  Correct use of Github Flow
  ✅  No linter errors
  ✅  Descriptive Pull Request

## Required Changes ♻️

#### [YOUR DESIGN]
Screenshot 2022-08-27 at 12 59 14 PM

Screenshot 2022-08-27 at 1 00 30 PM


#### [REQUIRED DESIGN]
Screenshot 2022-08-27 at 12 59 44 PM

- [ ] Please add a little a bit padding to the project card so that title, description will become easy to read 😃 
- [ ] See project button is stick to the bottom in the project card in the required Figma design. Kindly remove the padding/margin right below your see project button to make it look as similar as possible. 😊 

- [ ] REQUIRED DESIGN

Screenshot 2022-08-27 at 1 03 45 PM

- [ ] YOUR DESIGN
Screenshot 2022-08-27 at 1 04 15 PM


_Check the comments under the review._




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (prg-04) [reply index.html]: I can't find a ballot box entities that I can style according to the template but I've found one close to the design

File: index.html [hunk 3]
```diff
@@ -13,40 +13,219 @@
...
+              </ul>
+            </div>
+            <button>See Project</button>
+          </div>
+        </div>
+      </section>
+      <section class="about">
+        <div class="about-me">
+          <h2>About Me</h2>
+          <p>
```
Comment (tufoinnkuo10) [inline index.html:188]: - [ ] great job making some of the changes. Please use the correct font size and weight. also add the button size and text size. like the previous reviewer noted please adjust the width. Nice work
- [ optional] the about me background logo is below the button and the line under it. please look into that
-your design 
Screenshot (350)_LI
figma
Screenshot (351)

File: index.html [hunk 4]
```diff
@@ -13,40 +13,219 @@
...
+            hesitate to contact me.
+          </p>
+          <button>Get My Resume</button>
+        </div>
+        <hr />
+        <div class="skills">
+          <div class="language">
+            <h3 class="head"><span>&#9670;</span> Language</h3>
+            <ul>
+              <li>Javascript</li>
```
Comment (tufoinnkuo10) [inline index.html:201]: - [ ] please like the previous reviewer noted, kindly align this section well. u can centralize the `li`
yours
Screenshot (351)_LI
required
Screenshot (353)

File: index.html [hunk 5]
```diff
@@ -13,40 +13,219 @@
...
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
+              </p>
+              <ul>
+                <li><a href="#">html</a></li>
+                <li><a href="#">bootstrap</a></li>
+                <li><a href="#">Ruby</a></li>
+              </ul>
+            </div>
+            <button>See Project</button>
```
Comment (tufoinnkuo10) [inline index.html:151]: - [ optional]  please make the button text color white, increase the font size as well.

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @prg-04 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html [hunk 6]
```diff
@@ -13,40 +13,219 @@
...
+          </div>
+        </div>
+      </section>
+      <section class="about">
+        <div class="about-me">
+          <h2 class="h2">About me</h2>
```
Comment (BenMukebo) [inline index.html:187]: - [OPTIONAL] As requested by the previous reviouwer, Kindly make sure that the logo is below the button and the line under it. please look into the original design 👇🏻 

- Figma design:

image

File: index.html (hunk 4 above)
Comment (BenMukebo) [reply index.html:201]: > * [ ]  please like the previous reviewer noted, kindly align this section well. u can centralize the `li`
>   yours
>   Screenshot (351)_LI
>   required
>   Screenshot (353)

- [ ] Kindly as requested by the previous reviewer, make sure you address this 🙃

**Review by BenMukebo — CHANGES_REQUESTED** [review]
Hi @prg-04, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To Highlight :clap: 
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️


 ## Required Changes ♻️
_Check the comments under the review._


You have submitted a project, that Linters check are not running in the GitHub/action
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please Tag me on (@BenMukebo) when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @prg-04 🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- No Linter errors ✅
- Your repo is professional ✅
- Well structured files ✅



Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**